content-based-message-routing/README.md

```diff
@@ -0,0 +1,496 @@
+# Content-based message routing
+
+## Overview
+
+In this tutorial, you will develop an appointment reservation service for several hospitals. Requests are routed to the relevant hospital based on the content of the request payload.
+
+To implement this use case, you will develop a REST service with a single resource using Visual Studio Code with the Ballerina Swan Lake extension. The resource will receive the user request, select the hospital service based on `hospital_id`, send a request to the hospital service, and respond with the correct reservation details.
+
+The flow is as follows:
+
+1. Receive a request with a JSON payload similar to the following.
+
+```json
+{
+    "patient": {
+        "name": "John Doe",
+        "dob": "1940-03-19",
+        "ssn": "234-23-525",
+        "address": "California",
+        "phone": "8770586755",
+        "email": "johndoe@gmail.com",
+        "card_no": "7844481124110331"
+    },
+    "doctor": "thomas collins",
+    "hospital": "grand oak community hospital",
+    "appointment_date": "2017-04-02"
+}
+```
+
+2. Extract `hospital_id` field and select the hospital service endpoint according to the following criteria. 
```

Comment:
Mentioned previously also, but no `hospital_id` in the sample payload?

Reply:
got it.